### PR TITLE
AWS Cluster create

### PR DESCRIPTION
**Pull request by brennancheung**

### CODE REVIEW

**Comment by brennancheung** [comment]
All the fields seem to be working.  I'll add the submit functionality and test that it is working on a real server in another PR.

File: src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js [hunk 1]
```diff
@@ -144,9 +119,100 @@ const handleNetworkPluginChange = ({ setWizardContext, setFieldValue }) => optio
...
 // These fields are only rendered when the user opts to not use a `platform9.net` domain.
-const renderCustomNetworkingFields = ({ params, values }) => {
-  const updateFqdns = () => {
-    // TODO: When the domain changes, update the API and services FQDN
+const renderCustomNetworkingFields = ({ params, getParamsUpdater, values, setFieldValue, setWizardContext, wizardContext }) => {
+  const updateFqdns = (value, label) => {
+    console.log('wizardContext')
+    console.log(wizardContext)
+    console.log('values')
+    console.log(values)
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js]: console.logs here

File: src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js [hunk 2]
```diff
@@ -155,46 +221,65 @@ const renderCustomNetworkingFields = ({ params, values }) => {
...
+  if (data.httpProxy) { body.httpProxy = data.httpProxy }
+  if (data.networkPlugin === 'calico') { body.mtuSize = data.mtuSize }
+
+  data.runtimeConfig = {
+    default: '',
+    all: 'api/all=true',
+    custom: data.customRuntimeConfig,
+  }[data.runtimeConfigOption]
+
+  console.log('TODO: submit API call with body')
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js]: console.logs here

File: src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js [hunk 3]
```diff
@@ -155,46 +221,65 @@ const renderCustomNetworkingFields = ({ params, values }) => {
...
       />
-      {renderVpcFields(values.vpcOption, {})}
+      {renderNetworkFields(values.network)}
     </>
   )
 }
 
-const handleSubmit = () => {
-  // TODO
+const handleSubmit = params => data => {
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js]: we could memoize this so we don't recreate a new function on each render unless a param has changed

**Pull request merged**